website/docs/add-secure-apps/flows-stages/bindings/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Bindings
+---
+
+A binding is, simply put, a connection between two components (a flow, stage, policy, user, or group) _that adds additional content_ to one those existing components.
```

Comment:
"Content" is such a weird word.  "Functionality?" "Intelligence?"

Reply:
I agree, content is not at all the right word here. Let me try to reword...